tests/Functional/App/cases/Workflow/config.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns:content-api="http://libero.pub/schema/content-api-bundle"
+    xmlns:framework="http://symfony.com/schema/dic/symfony"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://symfony.com/schema/dic/services"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <framework:config>
+        <framework:workflow name="libero.content_api.put" type="state_machine">
```

Comment:
Not sure how to handle this configuration yet. The basics will be fixed, but the places and transitions (between `needs_validation ` and `end` can be custom.)

Reply:
Will ignore for now.